<?xml version="1.0" encoding="utf-8"?>

<!--
     This version does not set the sphere_obstacle_constraint_csv parameter and uses 
     the version already explicitly set in the parameter server

  <arg name="sphere_obstacle_constraint_csv" default="0.0,0.5,0.1,0.15"/>
-->

<launch>
  <arg name="x_target_x" />
  <arg name="x_target_y" />
  <arg name="x_target_z" />
  <arg name="behavioral_type" default="" />

  <arg name="upright_constraint_direction_csv" default="" />
  <arg name="use_upright_orientation_constraint" default="false" />
  <arg name="use_upright_orientation_constraint_end_only" default="false" />

  <arg name="approach_constraint_csv" default="" />
  <arg name="shape_approach" default="false" />

  <arg name="passthrough_constraint_csv" default="" />
  <arg name="obstacle_linearization_constraint_csv" default="" />

  <param name="x_target_x" value="$(arg x_target_x)" />
	<param name="x_target_y" value="$(arg x_target_y)" />
	<param name="x_target_z" value="$(arg x_target_z)" />
  <param name="behavioral_type" value="$(arg behavioral_type)" />

  <param name="upright_constraint_direction_csv" value="$(arg upright_constraint_direction_csv)" />
  <param name="use_upright_orientation_constraint" value="$(arg use_upright_orientation_constraint)" />
  <param name="use_upright_orientation_constraint_end_only" value="$(arg use_upright_orientation_constraint_end_only)" />

	<param name="approach_constraint_csv" value="$(arg approach_constraint_csv)" />
  <param name="shape_approach" value="$(arg shape_approach)" />

	<param name="passthrough_constraint_csv" value="$(arg passthrough_constraint_csv)" />
  <param name="obstacle_linearization_constraint_csv" value="$(arg obstacle_linearization_constraint_csv)" />

  <node name="mico_simple_move_client" 
    pkg="nw_mico_client" 
    type="nw_mico_simple_move_client" 
    clear_params="true"
    output="screen">		
  </node>
</launch>
